ts/Series/Map/MapSeries.ts

```diff
@@ -682,6 +682,49 @@ class MapSeries extends ScatterSeries {
 
             // Add class names
             this.points.forEach((point): void => {
+                if (point.graphic) {
```

Comment:
We already had an `if (point.graphic) {` below, so we should just combine them. And use the local `graphic` variable below too.